en/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md

```diff
@@ -0,0 +1,159 @@
+
+1. Install the Centreon Plugin on every poller monitoring Nginx Plus resources: :
+
+```bash
+yum install centreon-plugin-Applications-Nginx-Plus-Restapi.noarch
+```
+
+2. On the Centreon Web interface in "Configuration > Plugin packs > Manager", install the *Nginx Plus* Plugin-Pack
+
+<!--Offline IMP License-->
+
+1. Install the Centreon Plugin on every poller monitoring Nginx Plus resources :
+
+```bash
+yum install centreon-plugin-Applications-Nginx-Plus-Restapi.noarch
+```
+
+2. On the Centreon Central server, install the Centreon Plugin-Pack from the RPM:
+
+```bash
+yum install centreon-pack-applications-nginx-plus-restapi.noarch
+```
+
+3. On the Centreon Web interface in "Configuration > Plugin packs > Manager", install the *Nginx Plus* Plugin-Pack
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+This Plugin-Pack is designed to have in Centreon one host per Nginx Plus environment.
+Adding a host into Centreon, link it to the template named *App-Nginx-Plus-Restapi-custom*.
+Once the template applied, some Macros have to be configured:
+| Mandatory | Name        | Description                                      |
+| :-------- | :---------- | :----------------------------------------------- |
+| X         | APIPORT     | Port used (Default: 443)                         |
+| X         | APIPROTO    | Specify https if needed (Default: 'https')       |
+| X         | APIUSERNAME | Nginx basic username                             |
+| X         | APIPASSWORD | Nginx basic password.                            |
+| X         | APIPATH     | Specify api path (Default: '/api/6')             |
+
+
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the Plugin installed, log into your poller using the *centreon-engine* user account and test 
+by running the following command (Parameters such as ```api-username``` or ```api-path``` have to be adjusted):
+
+```bash
+/usr/lib/centreon/plugins/centreon_nginx_plus_restapi.pl \
+    --plugin=apps::nginx::nginxplus::restapi::plugin \
+	--mode=connections \
+	--port='443' \
+	--proto='https' \
+	--api-username='myapiuser' \
+	--api-password='myapipassword' \
+	--api-path='/api/6' \
+	--filter-counters='mycountersfilter' \
+	--warning-active='60' \
+	--critical-active='80'
+	--warning-idle='8' \
+	--critical-idle='10' \
+    --warning-accepted='50' 
+	--critical-accepted='65' \
+	--warning-dropped='3' \
+    --critical-dropped='5' \
+	--verbose
+	
+
+OK: Active : 5, Idle : 0, Accepted : 5, Dropped : 0|
+'connections.active.count'=5;;60;80; 'connections.idle.count'=1;;8;10; 'connections.accepted.count'=5;;50;65; 'connections.dropped.count'=0;;3;5;
+```
+
+The above command controls the Nginx Plus connections (```--mode=connections```).
+This command will trigger a WARNING alarm if the active connections increase to 60 (```--warning-active='60'```)
+and a CRITICAL alarm if it increases to 80 (```--critical-active='80'```).

```

Comment:
Enhance the description of the command (mode/plugin path/).

Reply:
Added description of the command